mastering-plone/behaviors_1.md

```diff
@@ -2,8 +2,17 @@
 
 # Behaviors
 
-````{sidebar}
-Get the code! ({doc}`More info <code>`)
+````{sidebar} Classic chapter
+```{figure} _static/plone.svg
+:alt: Plone Logo
+```
+
+This chapter is about Plone Classic.
+
+---
+
+**Get the code! ({doc}`More info <code>`)**
```

Comment:
Where is "more info"? Is this a placeholder or todo?

Reply:
Actually I see that occurs multiple times. I'm not sure whether this is finished, but it looks strange to me.

Reply:
This is converted fine, just as it should be. It's a refined way to reference documents. https://jupyterbook.org/reference/cheatsheet.html#reference-documents The target of the link is page http://localhost:3004/mastering-plone/code.html

Reply:
OK, now I see it. I misinterpreted `<code>` as an HTML tag, not as a target.